tests/test-cluster.py

```diff
@@ -87,7 +98,9 @@ def _setup_lxc(self, channel_or_snap, launch_configs=None):
                 ).split()
             )
             time.sleep(20)
-            self._load_launch_configuration_lxc(launch_configs)
+            if is_ipv6_configured:
+                self._load_launch_configuration_lxc(launch_configs)
```

Comment:
I also need to point out a blind spot you have. There is a second backend, multipass.

Reply:
Yeah but that is never triggered: https://github.com/canonical/microk8s/blob/6f488f61794fdd0ae30f626c8810acbfd0274768/tests/test-cluster.py#L188C9-L188C45. I'll fix this too